tests/unit/mocks.py

```diff
@@ -85,36 +84,6 @@ def valid(self) -> bool:
         return self.token is not None and not self.expired
 
 
-class BadRefresh(Exception):
-    pass
-
-
-class MockMetadata(ConnectionInfo):
-    """Mock class for ConnectionInfo"""
-
-    def __init__(
-        self, expiration: datetime.datetime, ip_addrs: Dict = {"PRIMARY": "0.0.0.0"}
-    ) -> None:
-        self.expiration = expiration
-        self.ip_addrs = ip_addrs
-
-
-async def instance_metadata_success(*args: Any, **kwargs: Any) -> MockMetadata:
-    return MockMetadata(
-        datetime.datetime.now(datetime.timezone.utc) + datetime.timedelta(minutes=10)
-    )
-
-
-async def instance_metadata_expired(*args: Any, **kwargs: Any) -> MockMetadata:
-    return MockMetadata(
-        datetime.datetime.now(datetime.timezone.utc) - datetime.timedelta(minutes=10)
-    )
-
-
-async def instance_metadata_error(*args: Any, **kwargs: Any) -> None:
-    raise BadRefresh("something went wrong...")
-
-
```

Comment:
this legacy mock was hiding a bad test, removed it and now our tests properly rely on our internals.